Cargo.toml

```diff
@@ -13,13 +13,16 @@ keywords = ["fft"]
 [dependencies]
 num-complex = "0.4"
 dyn-stack = { version = "0.8", default-features = false }
+pulp = "0.10"
+bytemuck = "1.13"
```

Comment:
these would need to be optional right and only used when fft128 is enabled ?

Reply:
what's the point of making them optional?

Reply:
they are only used by fft128 right ?

Reply:
yeah

Reply:
so IIRC the pattern people tend to follow is to mark dependencies as optional and only enable them when a certain feature is enabled to reduce the amount of code being compiled, so here as it's fft128 only I would tie the dependencies being active to the fft128 feature being active as well

Reply:
one problem i'm thinking of in that case is that `pulp/nightly` and `bytemuck/nightly-stdsimd` should only be enabled if both `fft128` and `nightly` are enabled. how do we express that in Cargo.toml?

Reply:
it looks like we can use this

https://doc.rust-lang.org/cargo/reference/features.html#dependency-features

Reply:
that looks great, thanks a lot !